src/frontend/post-github-setup.ts

```diff
@@ -25,53 +26,51 @@ const renderGitHubSetupPageVersion = async (
 	res.render("github-setup.hbs", gitHubSetupPagePayload)
 };
 
-const validateJiraSite = (
-	jiraSiteUrl: string,
+const validateJiraSite = async (
+	jiraHost: string,
 	domain: string,
 	req: Request,
 	res: Response
-): void => {
-	// Check that the entered domain is valid by making a request to the status endpoint
-	axios(`${jiraSiteUrl}/status`, {
-		method: "GET",
-		headers: {
-			"content-type": "application/json",
-		},
-	})
-		.then((response) => {
-			// If Jira site is valid, response returns a state of RUNNING
-			if (response?.data?.state === "RUNNING") {
```

Comment:
Wondering why we have removed this check?

Reply:
@mboudreau ?

Reply:
First, because the false conditional did nothing and just returned a blank page to the user, and second, we don't need to as if we call `/status` on site that doesn't exist, it returns a 404.  I couldn't find much documentation on this endpoint, but I'm guessing that it returns a `running: false` only if it's been disabled or in maintenance mode of some kind.  Not sure if/how we want to handle that edge case.